draft-ietf-tls-dtls13.md

```diff
@@ -1607,9 +1607,17 @@ After the handshake, ACKs SHOULD be sent once for each received
 and processed handshake record (potentially subject to some delay) and MAY
 cover more than one flight.
 
-ACK records MUST be sent with an epoch that is equal to or higher
-than the record which is being acknowledged. Implementations SHOULD
-simply use the current key.
+During the handshake, ACK records MUST be sent with an epoch that is
+equal to or higher than the record which is being acknowledged.
```

Comment:
This is a bit pedantic, but strictly speaking this would allow a Client to ACK a ServerHello with epoch 1 (early data) - this doesn't seem right, because the server might not accept it.

Reply:
Sorry, I had QUIC epochs in my head. Will fix.